content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -330,21 +320,17 @@ clusters, zones, or regions.
 This model affords the following benefits:
 
 - Improved availability: If a control plane becomes unavailable, the scope of
-  the outage is limited to only that control plane.
+  the outage is limited to only workloads in clusters managed by that control plane.
 
 - Configuration isolation: You can make configuration changes in one cluster,
   zone, or region without impacting others.
 
-- Controlled Rollout: You have more fine-grained control over configuration
-  rollout (e.g. one cluster at a time). For large meshes with many clusters,
-  this process can be automated with CI/CD pipelines.
+- Controlled rollout: You have more fine-grained control over configuration
+  rollout, for example, to canary configuration changes in a sub-section of the mesh
+  controlled by a given primary cluster.
 
-- Selective Service Visibility: You can choose which `Service`/`ServiceEntry`
-  resources to make visible to each control plane. This can be used to
-  establish service-level isolation between clusters. For example, an
-  administrator may choose to deploy the `HelloWorld` service to Cluster A,
-  but not Cluster B. Any attempt to call `HelloWorld` from Cluster B will
-  fail the DNS lookup.
+- Selective service visibility: You can restrict service visibility to part
```

Comment:
Why did you remove the example in this bullet?

Reply:
I thought its TMI for this doc, but can add it back if you feel strongly about it.

Reply:
It helps to illustrate what we mean by `restricting service visibility`, I think it's probably useful.

Reply:
added